lib/task_helpers/payload_player.rb

```diff
@@ -0,0 +1,77 @@
+class PayloadPlayer
+  POST_HEADERS = { 'Content-Type' => 'application/json' }.freeze
+  GET_ASSESSMENT_REGEX = %r{^/assessments/[0-9a-f-]{36}$}.freeze
+  POST_ASSESSMENT_REGEX = %r{^/assessments/([0-9a-f-]{36})}.freeze
+  SERVER_HOST = 'http://localhost:4000'.freeze
+
+  def self.call
+    new.call
+  end
+
+  def initialize
+    @yaml = YAML.load_file(Rails.root.join('tmp/api_replay.yml'))
```

Comment:
Where is tmp/api_replay.yml created ? I'm assuming an error will occur saying no such directory.

Reply:
I just saw that in the [Apply PR ](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/2085/files)it says to copy and paste tmp/api_replay.yml into the CFE repo : Z

Could this be updated in the README or in confluence. 

Maybe do puts '>>> Get the tmp/api_replay.yml file from Apply using ...' 
This would create a dependency though, between apply and CFE unless this rake task is just temporary?